### PR TITLE
OSDOCS-3073: Adds RN for cluster support

### PR DESCRIPTION
**Pull request by opayne1**
Applies to [OSDOCS-3073](https://issues.redhat.com/browse/OSDOCS-3073)

- Applies to `enterprise-4.10` only
- [Preview link](url)
- Adding known issue based on [BZ2048059](https://bugzilla.redhat.com/show_bug.cgi?id=2048059)

### CODE REVIEW

**Comment by opayne1** [comment]
@kdoberst PTAL. Thank you!

**Comment by opayne1** [comment]
@yapei PTAL. Thank you!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -135,6 +135,11 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-cluster-support-level"]
+==== Cluster support level
+
+{product-title} now allows you to easily view support level information about your cluster via an alert on the *Overview* -> *Cluster* page, in the *Cluster Settings*, in the *About* modal, and adds a notification to your notifications drawer when the cluster is unsupported. You can manage subscription settings under the *Service Level Agreement(SLA)* of the *Overview* page.
```
Comment (yapei) [inline release_notes/ocp-4-10-release-notes.adoc:141]: `an alert on the *Overview* -> *Cluster* page` , we show SLA info in Home -> Overview -> Details card
`in the *Cluster Settings*`, shall we use `in Cluste Settings page` or `on Cluster Settings page` ? 
attaching a screenshot for reference
Screen Shot 2022-02-17 at 3 31 57 PM

Comment (yapei) [reply release_notes/ocp-4-10-release-notes.adoc:141]: `You can manage subscription settings under the *Service Level Agreement(SLA)* of the *Overview* page`  Shall we update it to something like `You can manage subscription settings by clicking `Manage subscription settings` link under the *Service Level Agreement(SLA)* of the *Overview* page`

Comment (yapei) [reply release_notes/ocp-4-10-release-notes.adoc:141]: also we have SLA info can not be shown correctly due to bug https://bugzilla.redhat.com/show_bug.cgi?id=2048059

Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:141]: I think there's a missing space between Agreement and (SLA).

**Comment by kdoberst** [comment]
Looks good to me

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -135,6 +135,11 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-cluster-support-level"]
+==== Cluster support level
+
+{product-title} now allows you to easily view support level information about your cluster via an alert on the *Overview* -> *Cluster* page, in the *Cluster Settings*, in the *About* modal, and adds a notification to your notifications drawer when the cluster is unsupported. You can manage subscription settings by clicking *Manage Subscription Settings* under the *Service Level Agreement (SLA)* of the *Overview* page.
```
Comment (yapei) [inline release_notes/ocp-4-10-release-notes.adoc:141]: I'm afraid we don't have `Overview -> Cluster page`, are you targetting `Overview -> Details card` or something else?

**Comment by yapei** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=2048059 is a bug we should add to `Known issues`

File: release_notes/ocp-4-10-release-notes.adoc (hunk 2 above)
Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc]: @yapei yes, I think so. I have updated and moved to the known issues section of the release notes.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -1116,6 +1116,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 
 
+* Support level information about your cluster via an alert on the *Home* -> *Overview* -> *Details* card currently indicates the *Service Level Agreement (SLA)* is `unknown`. It will be fixed in a future version of {product-title}. For more information, see (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048059[*BZ#2048059]).
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:1119]: Perhaps make "Currently, support level . . . " the introductory clause
I suggest dropping "via an alert" to improve clarity.
Maybe instead of "It" we can say "The incorrect status" 
You can remove the parentheses and formatting around the BZ#2048059 link.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -1114,7 +1114,7 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * The ability to delete operands from the web console using the `delete all operands` option is currently disabled. It will be re-enabled in a future version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2012120[BZ#2012120] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2012971[BZ#2012971].
 
-
+* Currently, Support level information about your cluster on the *Home* -> *Overview* -> *Details* card currently indicates the *Service Level Agreement (SLA)* is `unknown`. The incorrect status will be fixed in a future version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2048059[*BZ#2048059].
```
Comment (yapei) [inline release_notes/ocp-4-10-release-notes.adoc:1117]: so we have only this line to be added for cluster support in RH, right?

Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:1117]: That is correct.

**Comment by opayne1** [comment]
@yapei moved back to "new features and enhancements" section based on https://bugzilla.redhat.com/show_bug.cgi?id=2057438#c3. Is this note ready to go?

**Comment by yapei** [comment]
@opayne1 thanks for your continuous updates, it looks good to me!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 5]
```diff
@@ -194,6 +194,11 @@ With this update, you can customize workload notifications on the *User Preferen
 ==== Improved quota visibility
 With this update, non-admin users are now able to view their usage of the `AppliedClusterResourceQuota` on the *Project Overview*, *ResourceQuotas*, and *API Explorer* pages in order to determine the cluster-scoped quota available for use. Additionally, `AppliedClusterResourceQuota` details can now be found on the *Search* page.
 
+[id="ocp-cluster-support-level"]
+==== Cluster support level
+
+{product-title} now allows you to easily view support level information about your cluster on the *Overview* -> *Details* card, in the *Cluster Settings*, in the *About* modal, and adds a notification to your notifications drawer when the cluster is unsupported. You can manage subscription settings under the *Service Level Agreement (SLA)* of the *Overview* page.
```
Comment (sagidlow) [inline release_notes/ocp-4-10-release-notes.adoc:200]:
```suggestion
{product-title} now enables you to view support level information about your cluster on the *Overview* -> *Details* card, in the *Cluster Settings*, in the *About* modal, and adds a notification to your notifications drawer when the cluster is unsupported. You can manage subscription settings under the *Service Level Agreement (SLA)* of the *Overview* page.
```

- From the IBM Style Guide on "allow"

> allow   verb
> Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."

-  "Easy" is a subjective term, I would try to avoid it. IBM Style Guide says:

> Use with caution. It is normally not accurate to say, “With IBM's service products, installation of your new POWER system will be easy.” It may be accurate to say, “IBM's service products can be used to make installation of your new POWER system easier than if you tried to manage your own installation.”

- The first sentence is really hard to follow, in my opinion. I would consider something like: 

"{product-title} now enables you to view support level information about your cluster. You can add a notification to your notifications drawer when the cluster is unsupported. To add a notification, navigate to *Overview* -> *Details* card in the *Cluster Settings*, in the *About* modal. From the Overview page, you can manage subscription settings under the *Service Level Agreement (SLA)*. "
